WooCommerce/Classes/Extensions/String+HTML.swift

```diff
@@ -42,4 +42,16 @@ extension String {
         let string = replacingOccurrences(of: "<[^>]+>", with: "", options: .regularExpression, range: nil)
         return string
     }
+
+    /// Indicates whether the string contains HTML content.
+    ///
+    /// This property uses the `strippedHTML` method to determine if the string contains HTML.
+    /// If the stripped version is different from the original string, it's considered to contain HTML.
+    ///
+    /// NOTE: This can be slow ⏳ — using it in a background thread is strongly recommended.
+    ///
+    var containsHTML: Bool {
+        let stripped = self.strippedHTML
+        return stripped != self
+    }
```

Comment:
@hichamboushaba and I discussed back and forth on this whether to display Aztec automatically or not when the content is HTML, and this function is meant to support that.

This function is not yet used in any part of the PR since the latest decision is to show a button to show Aztec separately, but might be needed later on. It can also be deleted later once we finalize everything.

Reply:
With the latest conversation I think we want to make aztec editor available to all kind of text, and let user choose to use it instead of automatically selecting it for them, so this becomes unnecessary anymore. Removed in https://github.com/woocommerce/woocommerce-ios/pull/13863/commits/45624f2a74c1ae8be56c43aac1798424a9195fbf